### PR TITLE
Feature/badge

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added styling for icon and avatar badge.

### CODE REVIEW

File: css/components/avatar/index.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	width: 3rem;
+	height: 3rem;
+
+}
+.avatar-badge-container{
+position: relative;
+/* transform:scale(1);
+    border-radius: 50%;
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
```
Comment (DivyanshSareen) [inline css/components/avatar/index.css:77]: remove comment if not intended for future use

**Review by DivyanshSareen — APPROVED** [review]
many extra spaces in /alert/index.css, remove them and you are good to merge 👍

**Comment by Sahil25061999** [comment]
As suggested by @DivyanshSareen remove the extra spaces

File: css/components/avatar/index.css (hunk 1 above)
Comment (sumitk-singh) [reply css/components/avatar/index.css:77]: +1 @DivyanshSareen

**Review by sumitk-singh — APPROVED** [review]
all looks good just remove the commented code.

File: css/components/avatar/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/components/avatar/index.css:77]: okay

**Pull request merged**